daprdocs/content/en/reference/daprd-reference.md

```diff
@@ -0,0 +1,38 @@
+---
```

Comment:
- [ ] Please nest this page under a directory named `dapr-services`.
- [ ] Add an `_index.md` dile under `dapr-services` with front matter describing it.

Reply:
@orizohar I see this conflicts with your feedback, but I think we should add a "Dapr services" folder so we can add additional references on dapr-placement, dapr-sidecar-injector, and other Dapr services.